colorbleed/plugins/fusion/load/actions.py

```diff
@@ -0,0 +1,103 @@
+"""A module containing generic loader actions that will display in the Loader.
+
+"""
+
+from avalon import api
+
+
+def _set_frame_range(start, end, set_render_range=True):
+    """Set Fusion comp's start and end frame range
+
+    Attrs:
+        set_render_range (bool, Optional): When True this will also set the
+            composition's render start and end frame.
+
+    Returns:
+        None
+
+    """
+
+    from avalon.fusion import get_current_comp, comp_lock_and_undo_chunk
+
+    comp = get_current_comp()
+
+    attrs = {
+        "COMPN_GlobalStart": start,
+        "COMPN_GlobalEnd": end
+    }
+
+    if set_render_range:
+        attrs.update({
+            "COMPN_RenderStart": start,
+            "COMPN_RenderEnd": end
+        })
+
+    with comp_lock_and_undo_chunk(comp):
+        comp.SetAttrs(attrs)
+
+
+class FusionSetFrameRangeLoader(api.Loader):
+    """Specific loader of Alembic for the avalon.animation family"""
+
+    families = ["colorbleed.animation",
+                "colorbleed.camera",
+                "colorbleed.imagesequence",
+                "colorbleed.yeticache",
```

Comment:
Yeti Cache has no place in Fusion, point cache is for future implementation if it ever is available in the API to load.

Reply:
Ah, no - this isn't actually loading the cache. It solely allows you to "take" the frame range from the yeti cache and apply it to the fusion scene. It's a very generic action, it should support anything that has a `startFrame` and `endFrame` data. So this is actually correct.

Reply:
Also note, pointcache is actually used in production. :+1:

Reply:
Yes it is used in production but I am assuming you do not need to line up your comp length based on a pointcache which is rendered to an image sequence which is loaded in your comp? :thinking: